.github/workflows/selfhosted.yml

```diff
@@ -77,7 +77,7 @@ jobs:
         source testenv/bin/activate
         export QIBOLAB_PLATFORMS=.
         partition="$(jq -r -n --argjson data "$queues" '$data.'$platform)"
-        srun -p $partition python selfhosted/calibration.py ${{ matrix.platform }}
+        QIBOCAL_LOG_LEVEL=4 QIBO_LOG_LEVEL=3 srun -p $partition python selfhosted/calibration.py ${{ matrix.platform }}
```

Comment:
Are the log levels still needed? I think we were using them to debug RB.

Reply:
Yeah, I already removed them when renaming the files

Reply:
Ok, I saw the the conversation was unsolved and I decided to reply.. a week later. I will resolve it now.